### PR TITLE
Feature/player page

### LINKED ISSUE #20: Add personal player pages
We should add views for individual players. These can live under `/players/<player_id>`. These should include player game histories and statistics.

### PR DESCRIPTION
**Pull request by hwchen**
Add Player pages.
closes #20 

Links to player pages added to player tables in dashboard, as well games table in games page and player page.

Sorry about the messy git history. I made a mistake with git commit --amend, but now there's two merge commits? If you care, I can go back and figure it out... but I think it will take a while.

### CODE REVIEW

**Comment by hndrewaall** [comment]
Hrm yeah. I usually avoid explicit merges and sync with master using rebases, tends to keep the history cleaner.

File: app/league/templates/dashboard/dashboard.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h2 class="sub-header">Players</h2>
           <tbody>
             {% for player in players -%}
             <tr>
-              <td>{{ player.aga_id }}</td>
+                <td><a href="players/{{player.id}}"> {{ player.aga_id }}</a></td>
```
Comment (hndrewaall) [inline app/league/templates/dashboard/dashboard.html]: What do you think of making the whole row, or at least the name, part of the link?

File: app/league/dashboard/views.py [hunk 2]
```diff
@@ -57,6 +57,15 @@ def create_player():
                            player_create_form=form)
 
 
+@blueprint.route('/players/<int:player_id>', methods=['GET'])
+@login_required
+def get_player(player_id):
+    """Get game history and statistics for player"""
+    player = Player.get_by_id(player_id)
+    games = player.games
```
Comment (hndrewaall) [inline app/league/dashboard/views.py]: Oops, forgot to post this:

Is it necessary to make this separate? Could easily reference `player.games` in the template.

File: app/league/templates/dashboard/dashboard.html (hunk 1 above)
Comment (hwchen) [inline app/league/templates/dashboard/dashboard.html]: Made the whole name linked. Whole row would require js. Maybe revisit later, and handle all the tables consistently.

File: app/league/dashboard/views.py (hunk 2 above)
Comment (hwchen) [inline app/league/dashboard/views.py]: change made

**Comment by hndrewaall** [comment]
I'm also thinking of putting AGA info (and future info) in its own little group. So, we'd have a little AGA pane, and then maybe a KGS or League Stats pane, etc. But! that can wait for another PR :)

**Comment by hndrewaall** [comment]
Looks good! gonna wait to see if @njritter has input, then we can merge

**Comment by njritter** [comment]
Awesome!

**Pull request merged**